crates/db_connection_pool/src/duckdbpool.rs

```diff
@@ -68,6 +71,32 @@ impl DuckDbConnectionPool {
 
         Ok(DuckDbConnectionPool { pool })
     }
+
+    /// Create a new `DuckDbConnectionPool` from data connector params.
+    ///
+    /// # Arguments
+    ///
+    /// * `params` - Data connector parameters for the connection pool.
+    ///
+    /// # Errors
+    ///
+    /// Returns an error if there is a problem creating the connection pool.
+    pub fn new_with_file_mode(params: &Arc<Option<HashMap<String, String>>>) -> Result<Self> {
+        let path = params
+            .as_ref()
+            .as_ref()
+            .and_then(|params| params.get("duckdb_file").cloned())
```

Comment:
I thought we wanted the yaml to be 

params:
  open: path/to/file.db

Reply:
Per the linked enhancement

Reply:
Updated